napari/utils/events/evented_model.py

```diff
@@ -79,21 +192,24 @@ class EventedMetaclass(main.ModelMetaclass):
 
     def __new__(mcs, name, bases, namespace, **kwargs):
         with no_class_attributes():
-            cls = super().__new__(mcs, name, bases, namespace, **kwargs)
+            with parametrized_generic_fix():
+                cls = super().__new__(mcs, name, bases, namespace, **kwargs)
```

Comment:
```suggestion
        with no_class_attributes(), parametrized_generic_fix():
            cls = super().__new__(mcs, name, bases, namespace, **kwargs)
```
can use two context managers with a comma